POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Views/FancyZonesPage.xaml.cs

```diff
@@ -23,41 +23,22 @@ namespace PowerToys_Settings_Sandbox.Views
     /// </summary>
     public sealed partial class FancyZonesPage : Page
     {
-        private bool hideTips = false;
         public FancyZonesPage()
         {
             this.InitializeComponent();
-
-            // if (!HaveExplainedAutoSave())
-            // {
-                   EnableFeatureTip.IsOpen = true;
-                    //AboutFeatureTip.IsOpen = true;
-            //     SetHaveExplainedAutoSave();
-            //}
         }
 
-        private void ShowAboutFeatureTip()
+        // This method toggles the image preview of the Module Overview link of the "About Feature" section
```

Comment:
Does this method really need a description?  Seems pretty obvious to me.  However, anytime you do want to comment a method, use the /// then tab key to build the proper structure.  In this case, I think its extra and not needed.

Reply:
Got it - thank you for the method comment tip!